outorg.el

```diff
@@ -1226,8 +1231,9 @@ source-block. Use `outorg-beginning-of-code' and
     (goto-char outorg-beginning-of-code)
 
     ;; Display blank line between text and block
-    (overlay-put (make-overlay (1- (point)) (1- (point)))
-                 'before-string "\n")
+    (when outorg-ensure-blank-line
+      (overlay-put (make-overlay (1- (point)) (1- (point)))
```

Comment:
The feature could also be improved by only inserting a blank line when necessary, which would avoid showing an extra blank line when one is already present.  e.g.:

```el
    ;; Display blank line between text and block
    (unless (looking-back "\n\n")
      (overlay-put (make-overlay (1- (point)) (1- (point)))
		   'before-string "\n"))
```

Reply:
I had considered this. Personally, this would be fine for my usage.

However, I still think we need to make this optional because even this is opinionated. I maintain that the default behavior should just be a vanilla conversion with no attempt at trying to guess how the user wants their blank lines displayed.

I would be fine with:
```elisp
;; Display blank line between text and block
(when org-ensure-blank-line
  (unless (looking-back "\n\n")
    (overlay-put (make-overlay (1- (point)) (1- (point)))
                 'before-string "\n")))
```

Reply:
That's fine with me as well.